### PR TITLE
Cleanup links

### PR DESCRIPTION
**Pull request by kprosise**
Set a number of redirected links to their current addresses. Checked a number of files for unused links, however some may remain. Older doc page links that were getting 404 errors have had redirects added.

Ran linkcheck. Rendered HTML and checked added redirects.

This commit addresses FFTK-140
This commit applies to MKT-429

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

what I hope are some small improvements to link related issues

## Checklist

* [ ] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

It is totally possible that some of the changed addresses are not desired. These were done in part to attempt to have a cleaner and maybe slightly more efficient linkcheck.  The redirects from older pages were included in a report from marketing.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 03ed8b89074eb73af8a6c8c6eeafe9000d85cef3 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2118/docs/artifacts/html/index.html

File: source/getting-started/install-fioctl/index.rst [hunk 1]
```diff
@@ -130,7 +130,7 @@ For this, you will generate OAuth2 application credentials for interacting with
 Adding Application Credentials
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Go to `Application Credentials <https://app.foundries.io/settings/credentials/>`_ and click on :guilabel:`+ New Credentials`.
+Go to `Application Credentials <https://app.foundries.io/login?next=%2Fsettings%2Fcredentials%2F>`_ and click on :guilabel:`+ New Credentials`.
```
Comment (MiloCasagrande) [inline source/getting-started/install-fioctl/index.rst:133]: You sure you want to change the app.fio addresses with the login + `next` url?
The previous one should work even if you are not logged in, it should redirect correctly to the `next` url.

Comment (kprosise) [reply source/getting-started/install-fioctl/index.rst:133]: Oh, right. I'll "undo" those ones.

**Review by angolini — APPROVED** [review]

File: source/user-guide/lmp-device-auto-register/lmp-device-auto-register.rst [hunk 2]
```diff
@@ -28,7 +28,7 @@ To follow this section, it is important to have:
 Creating Token
 --------------
 
-Go to `Tokens <https://app.foundries.io/settings/tokens/>`_ and create a new **Api Token** by clicking on
+Go to `Tokens <https://app.foundries.io/login?next=%2Fsettings%2Ftokens%2F>`_ and create a new **Api Token** by clicking on
```
Comment (kprosise) [inline source/user-guide/lmp-device-auto-register/lmp-device-auto-register.rst:31]:
```suggestion
Go to `Tokens <https://app.foundries.io/settings/tokens>`_ and create a new **Api Token** by clicking on
```

File: source/user-guide/submodule/submodule.rst [hunk 3]
```diff
@@ -265,7 +265,7 @@ Updating Submodule Automatically
 To automate the previous steps, you have to allow GitHub_ to access your 
 FoundriesFactory repository. For that, you need to create a token.
 
-Go to `Tokens <https://app.foundries.io/settings/tokens/>`_ and create a new **Api Token** by clicking on 
+Go to `Tokens <https://app.foundries.io/login?next=%2Fsettings%2Ftokens%2F>`_ and create a new **Api Token** by clicking on
```
Comment (kprosise) [inline source/user-guide/submodule/submodule.rst:268]:
```suggestion
Go to `Tokens <https://app.foundries.io/settings/tokens>`_ and create a new **Api Token** by clicking on 
```

**Review by kprosise — COMMENTED** [review]
suggestions

**Comment by doanac** [comment]
Docs for 0566d880ddd4aa2b0811a97aa7386820f7d259e2 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2122/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for c799ca51dbb95afb5670e76f84b75742b91d5db2 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2123/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for c7d520b15efe410d7d370bc1b7a86b14f8f46793 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2124/docs/artifacts/html/index.html

**Pull request merged**